### ISSUE TITLE
provide the calling graph of the processing in the log window

### ISSUE THREAD

**Issue by shelper** [dinusv/livecv#18]
It would be very useful if I tested the processing procedure and it works, now i want to add the processing procedure to my application source code, i can just open the log window, copy and paste to my source code editor. 
that will save alot of time for developing

**Comment by dinusv** [comment]
Hi, thank you for your suggestion, sounds interesting and it would be very useful for debugging as well. I would create a separate window for the call graph, the log would be filled with too much stuff in some cases.

I think we can manage this with a global CallGraph class instance that each algorithm calls just after it does the processing. We can store the matrix references there as well, so we can follow how each matrix gets transferred thorugh the calls.

**Comment by shelper** [comment]
i used your program a little bit more and i think it could be useful if the program separates edit window from the preview images, or support of 3rd part editors, like vim would be even better

 
i can open new issues if needed but i want just explain here first.

**Comment by dinusv** [comment]
True, I actually thought about using separate editors at some point, but never had the chance to look into it. There's also the preview mode which I want to add I think it falls in the same category.

I've created 2 issues from this: #19, #20.
Thanks for your input, you're welcome to open new issues if you have more suggestions
